src/index.html

```diff
@@ -6,7 +6,70 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
+  <body class="body">
     <h1>Calendar</h1>
+    <ul class="calendar calendar--start-day-sun calendar--month-length-31">
+      <li class="calendar__item"></li>
+
```

Comment:
remove extra empty lines between these childs `li`
```suggestion
```

Reply:
I know, but child tag in 1 line) So, would be better to remove these empty lines)